terraform/datadog/concourse.tf

```diff
@@ -52,6 +52,24 @@ resource "datadog_monitor" "continuous-smoketests-failures" {
   tags = ["deployment:${var.env}", "service:${var.env}_monitors"]
 }
 
+resource "datadog_monitor" "concourse-btrfs" {
+  name                = "${format("%s concourse excess btrfs processes running", var.env)}"
+  type                = "query alert"
+  message             = "Large number of btrfs processes running. Concourse probably needs shaking."
+  escalation_message  = "Large number of btrfs processes still running. Concourse probably needs shaking."
```

Comment:
I have no idea why we do it, and wondered the same - I was just keeping in line with other monitors.

[`escalation_message`](https://www.terraform.io/docs/providers/datadog/r/monitor.html#escalation_message) is optional, and I would happily remove it from both this and all the other cases where it seems to be needless.